spec/requests/api/v2/courses_spec.rb

```diff
@@ -0,0 +1,86 @@
+require "rails_helper"
+
+describe "Courses API v2", type: :request do
+  let(:user)         { create(:user) }
+  let(:organisation) { create(:organisation, users: [user]) }
+  let(:payload)      { { email: user.email } }
+  let(:token)        { build_jwt :apiv2, payload: payload }
+  let(:credentials) do
+    ActionController::HttpAuthentication::Token.encode_credentials(token)
+  end
+  let(:current_cycle) { find_or_create :recruitment_cycle }
+  let(:next_cycle)    { find_or_create :recruitment_cycle, :next }
+  let(:current_year)  { current_cycle.year.to_i }
+  let(:previous_year) { current_year - 1 }
+  let(:next_year)     { current_year + 1 }
+  let(:subjects) { [course_subject_mathematics] }
+
+  let(:providerA)       { create :provider, organisations: [organisation] }
```

Comment:
snake_case? not sure why the change to camelCase here?

Reply:
I changed it to use numbers since we seem to do it in other tests.